### PR TITLE
putAll method for objectmap

### PR DESCRIPTION
**Pull request by buthed010203**
I'm lazy

### CODE REVIEW

File: arc-core/src/arc/struct/ObjectMap.java [hunk 1]
```diff
@@ -183,6 +183,12 @@ public void putAll(ObjectMap<? extends K, ? extends V> map){
             put(entry.key, entry.value);
     }
 
+    public void putAll(Object... values) {
+        for(int i = 0; i < values.length / 2; i++){
+            put((K) values[i * 2], (V) values[i * 2 + 1]);
```
Comment (Anuken) [inline arc-core/src/arc/struct/ObjectMap.java:188]: redundant space after cast

Comment (buthed010203) [reply arc-core/src/arc/struct/ObjectMap.java:188]: ~~This was copy pasted from the code above~~

**Comment by buthed010203** [comment]
@Anuken fixed

File: arc-core/src/arc/struct/ObjectMap.java (hunk 1 above)
Comment (Anuken) [reply arc-core/src/arc/struct/ObjectMap.java]: well, then that's wrong too

**Pull request merged**